corehq/ex-submodules/casexml/apps/phone/caselogic.py

```diff
@@ -325,6 +325,15 @@ def batches(self):
                 batch = batch.next_batch
                 yield batch
 
+        yield UserCaseSyncCouchBatch(
+            self.user,
+            self.global_state,
+            self.domain,
+            self.last_synclog,
+            self.chunk_size,
+            case_sharing=self.case_sharing
+        )
```

Comment:
Should probably only do this if the domain has it enabled to avoid an extra couch request.